pkgs/desktops/gnome-3/extensions/gtile/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-shell-extension-gtile";
+  version = "34";
+
+  src = fetchFromGitHub {
+    owner = "gTile";
+    repo = "gTile";
+    rev = "V${version}";
+    sha256 = "1yka4acmrykh4vdd89yvn6yv89c6q16r72lgi3j48my3jghwjiwp";
+  };
+
+  uuid = "gTile@vibou";
+
+  nativeBuildInputs = [ glib ];
```

Comment:
Does not seem to be used.

Reply:
yeah, is see, glib is used only in `${glib.makeSchemaPath}` expression and there is no internal usage of glib (e.g.  glib-compile-schemas usage while Makefile target producing).